std/root.ln

```diff
@@ -638,7 +898,7 @@ export fn set(hm: HashMap<Hashable, any>, key: Hashable, val: any): HashMap<Hash
   const list = getR(hm.lookup[hash]);
   if list.length() == 8 {
     // Rebucket everything
-    const lookupLen = length(hm.lookup) * 2;
+    const lookupLen = length(hm.lookup) * 2 || 0;
     hm.lookup = new Array<Array<int64>> [ new Array<int64> [], ] * lookupLen;
```

Comment:
An unexpected impact of this is all of the other opcodes that take integers. Without unwrapping this integer you can't use it in the array construction line below. But if we allowed Result-wrapped values to be passed to the `repeat` function for arrays, it could fail besides OOM failure, and then you would need to unwrap the array.

Here it seems like adding Result-wrapped integer support would make things worse, not better, as the noise is unavoidable.

Reply:
I have already expressed my recent reservations about how eliminating most runtime errors make things more annoying and less usable. I only realized this after I started writing Alan in the standard library for the `Tree` type. Undoubtedly usability will improve once the compiler is smart enough to unwrap a Result within an  `isErr` or `isOk` conditional, but it is not clear to me by how much. As an individual developer, I would love to have the guarantee of most runtime errors impossible but it depends on the cost of usability I have to pay for it.

It also feels like it would not be the main reason someone would use Alan. This is true for me at least. I think we definitely want to add it at some point, but it doesn't feel correct to have it part of our MVP. Basically to me automatic parallelization >> most runtime errors impossible.